File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/#Teorico/0 - Revisao/5 - Comando IF/comand_IF.py
num = int(input("Digite um número: "))
if(num < 0):
    print("O número {0} é negativo!".format(num))

#Considerar 0 como nulo
elif(num == 0):
    print("O número {0} é nulo!".format(num))
else:     
    print("O número {0} é positivo!".format(num))

File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/#Teorico/0 - Revisao/5 - Comando IF/comand_IF.c
#include <stdio.h>

int main(int argc, char const *argv[]){
    int num;

    printf("Digite um número: ");
    scanf("%d", &num);

    if (num < 0){
        printf("Número %d é negativo!", num);
    }
    else{
        //Considerar 0 como nulo
        if (num == 0){
            printf("O número %d é Nulo!", num);
        }
        else{
            printf("O número %d é positivo!", num);
        }
    }
        
    return 0;
}
